frontend/app/views/main.scala.html

```diff
@@ -83,12 +83,16 @@
                 "@@type": "ListItem",
                 "position": 2,
                 "name":"@Html(schema.name)",
-                "item":[
-                @Html(Json.toJson(schema).toString)
-                ]
+                "item":[{
+                "@@type": "WebPage",
+                "id":"@Html(schema.url)"
+                }]
                 }]
                 }
             </script>
```

Comment:
One comment, it'd help with readability if the indentation for the json were consistent, eg. The opening `{` on line 70 is indented differently to the closing `}` on line 91.

Reply:
Adjusted indentation for the json